py/circuitpy_mpconfig.mk

```diff
@@ -109,6 +109,12 @@ CIRCUITPY_DIGITALIO = 1
 endif
 CFLAGS += -DCIRCUITPY_DIGITALIO=$(CIRCUITPY_DIGITALIO)
 
+ifndef CIRCUITPY_COUNTIO
+CIRCUITPY_COUNTIO = 1
```

Comment:
```suggestion
CIRCUITPY_COUNTIO = $(CIRCUITPY_FULL_BUILD)
```

Reply:
This will fix pyruler because it isn't a FULL_BUILD.

Reply:
Thanks that makes sense. I've now changed this.